jina/types/arrays/memmap.py

```diff
@@ -156,13 +170,13 @@ def append(self, doc: 'Document', flush: bool = True) -> None:
         if flush:
             self._header.flush()
             self._body.flush()
+        self._last_mmap = None
```

Comment:
since the invalidation happens in extend, we can invalidate just when flushing I think

Reply:
every operation that invalidates the cache should invalidate right?